Master

madrojudi opened the pull request.

Commit:
Dectection de charge

issotina committed:
Dectection de charge

maxoreli committed:
updated .idea directory

maxoreli committed:
Merging webservice resolve conflicts

maxoreli committed:
Merging Power branch resolve conflicts

Commit:
Updated idea directory

Commit:
Merge Master to map Resolve conflicts

issotina committed:
interfaces 1,2 et 3 ok

Commit:
Affichage sur la Maps les zones electrifiés ou non

Commit:
Merge branch 'webservice' of https://github.com/BeninDevelopers/ina-mobile into map

maxoreli committed:
Merging Power with Interfaces done

Commit:
Affichage sur la Maps les zones electrifiés ou non final

Commit:
fix the authors's names

maxoreli committed:
Merging of map branch ,resolve conflicts

maxoreli committed:
Merge branch 'map' of https://github.com/BeninDevelopers/ina-mobile

maxoreli committed:
Merge branch 'map' of https://github.com/BeninDevelopers/ina-mobile

maxoreli committed:
Integrating GCM Notif

joanef2 committed:
merging update master with GCM

Perempty committed:
Update README.md

Perempty committed:
Update README.md

Perempty committed:
Update README.md

Perempty committed:
Update README.md

fabricethilaw commented:
boolean mLoadingKeepGoing = true;
ProgressDialog mProgressDialog = new ProgressDialog(getActivity());
mProgressDialog.setProgressStyle(ProgressDialog.STYLE_SPINNER);
mProgressDialog.setTitle(R.string.progress_dialog_loading);
mProgressDialog.setCancelable(false);
mProgressDialog
        .setOnCancelListener(new DialogInterface.OnCancelListener() {
            @Override
            public void onCancel(DialogInterface dialog) {
                mLoadingKeepGoing = false;
            }
        });
 mProgressDialog.show();

On Sun, Feb 28, 2016 at 6:00 AM, madrojudi notifications@github.com wrote:

> ---
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/BeninDevelopers/ina-mobile/pull/1
> Commit Summary
> - Dectection de charge
> - Dectection de charge
> - updated .idea directory
> - Merging webservice resolve conflicts
> - Merging Power branch resolve conflicts
> - Updated idea directory
> - Merge Master to map Resolve conflicts
> - interfaces 1,2 et 3 ok
> - Affichage sur la Maps les zones electrifiés ou non
> - Merge branch 'webservice' of
>   https://github.com/BeninDevelopers/ina-mobile into map
> - Merging Power with Interfaces done
> - Affichage sur la Maps les zones electrifiés ou non final
> - fix the authors's names
> - Merging of map branch ,resolve conflicts
> - Merge branch 'map' of https://github.com/BeninDevelopers/ina-mobile
> - Integrating GCM Notif
> 
> File Changes
> - _A_ .idea/dbnavigator.xml
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-0
>   (449)
> - _M_ .idea/gradle.xml
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-1
>   (4)
> - _M_ .idea/misc.xml
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-2
>   (2)
> - _M_ .idea/vcs.xml
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-3
>   (1)
> - _M_ app/build.gradle
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-4
>   (9)
> - _A_ app/google-services.json
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-5
>   (42)
> - _M_ app/src/main/AndroidManifest.xml
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-6
>   (65)
> - _M_ app/src/main/java/org/benindevelopers/ina/MainActivity.java
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-7
>   (21)
> - _A_
>   app/src/main/java/org/benindevelopers/ina/MyGCMListenerService.java
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-8
>   (74)
> - _A_
>   app/src/main/java/org/benindevelopers/ina/MyInstanceIDListenerService.java
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-9
>   (40)
> - _A_
>   app/src/main/java/org/benindevelopers/ina/QuickstartPreferences.java
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-10
>   (24)
> - _A_
>   app/src/main/java/org/benindevelopers/ina/RegistrationIntentService.java
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-11
>   (108)
> - _A_ app/src/main/java/org/benindevelopers/ina/SearchActivity.java
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-12
>   (45)
> - _A_ app/src/main/java/org/benindevelopers/ina/Splashscreen.java
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-13
>   (41)
> - _A_ app/src/main/res/drawable-hdpi/app_icon.png
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-14
>   (0)
> - _A_ app/src/main/res/drawable-hdpi/ic_keyboard_voice_black_24dp.png
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-15
>   (0)
> - _A_ app/src/main/res/drawable-ldpi/app_icon.png
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-16
>   (0)
> - _A_ app/src/main/res/drawable-mdpi/app_icon.png
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-17
>   (0)
> - _A_ app/src/main/res/drawable-mdpi/ic_keyboard_voice_black_24dp.png
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-18
>   (0)
> - _A_ app/src/main/res/drawable-tvdpi/app_icon.png
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-19
>   (0)
> - _A_ app/src/main/res/drawable-xhdpi/app_icon.png
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-20
>   (0)
> - _A_ app/src/main/res/drawable-xhdpi/ic_keyboard_voice_black_24dp.png
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-21
>   (0)
> - _A_ app/src/main/res/drawable-xxhdpi/app_icon.png
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-22
>   (0)
> - _A_ app/src/main/res/drawable-xxhdpi/ic_keyboard_voice_black_24dp.png
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-23
>   (0)
> - _A_ app/src/main/res/drawable-xxxhdpi/app_icon.png
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-24
>   (0)
> - _A_
>   app/src/main/res/drawable-xxxhdpi/ic_keyboard_voice_black_24dp.png
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-25
>   (0)
> - _M_ app/src/main/res/layout/activity_main.xml
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-26
>   (4)
> - _A_ app/src/main/res/layout/activity_search.xml
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-27
>   (32)
> - _A_ app/src/main/res/layout/activity_splashscreen.xml
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-28
>   (56)
> - _A_ app/src/main/res/menu/search_menu.xml
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-29
>   (11)
> - _M_ app/src/main/res/values/colors.xml
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-30
>   (4)
> - _M_ app/src/main/res/values/dimens.xml
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-31
>   (1)
> - _M_ app/src/main/res/values/strings.xml
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-32
>   (4)
> - _M_ app/src/main/res/values/styles.xml
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-33
>   (10)
> - _M_ build.gradle
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-34
>   (3)
> - _M_ maps/build.gradle
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-35
>   (5)
> - _A_ maps/libs/osmdroid-android-4.3.jar
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-36
>   (0)
> - _A_ maps/libs/slf4j-android-1.6.1-RC1.jar
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-37
>   (0)
> - _A_ maps/src/main/java/org/benindevelopers/maps/Map.java
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-38
>   (116)
> - _A_ maps/src/main/res/drawable/green_marker.png
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-39
>   (0)
> - _A_ maps/src/main/res/drawable/red_marker.png
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-40
>   (0)
> - _A_ maps/src/main/res/layout/map.xml
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-41
>   (12)
> - _M_ power/src/main/AndroidManifest.xml
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-42
>   (7)
> - _A_
>   power/src/main/java/org/benindevelopers/power/PowerConnectionReceiver.java
>   https://github.com/BeninDevelopers/ina-mobile/pull/1/files#diff-43
>   (41)
> 
> Patch Links:
> - https://github.com/BeninDevelopers/ina-mobile/pull/1.patch
> - https://github.com/BeninDevelopers/ina-mobile/pull/1.diff
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/BeninDevelopers/ina-mobile/pull/1.

joanef2 committed:
- Intégration au module App des modules: Power, GCM, Webservices, Map
- Intégration de l'enrégistrement du smartphone dans le SplahScreenActivity
- Utilisation du Dialog natif Android pour les dialog de l'app (plutot que la librairie MaterialDialog)
- Mise à jour de la librairie OSMDroid à la version 5.1
- Intégration des appels webservices
- Affichages des points clickables sur le Map
- Bar de recherche avec suggestion des différentes zones prises en compte par la plateforme
- Affichage du dialog d'interpretation de l'état d'une zone au click sur un point de la carte et au click sur une suggestion du champ de recherche

joanef2 committed:
Merge remote-tracking branch 'origin/master'

joanef2 committed:
- Intégration de la détection automatique au module App
- Intégration du dialog de recupération de la source de courant utilisé quand le smartphone est branché en charge
- Amélioration du processus de détecion automatique de charge

joanef2 committed:
- Modification des appels WebService afin de prendre en compte le cas: "Nous n'avons pas assez d'info sur cette zone"
- Réduction du temps d'affichage du Splashscreen
- Affichage du nom de la zone sélectionnée lors de l'affichage de l'état d'une zone

joanef2 committed:
- Ajout de l'apk signé

joanef2 committed:
- Remplacement du logo de l'app par celle proposé par Vital Lokossou

joanef2 committed:
- Remplacement du logo de l'app par celui proposé par Vital Lokossou

maxxhount committed:
Correction de bugs et ajout de la page a propos

joanef2 committed:
- Mise à jour du gradle plugin
- Bug "L'application s'est arretée" lors de la détection de charge corrigé
- Desormais la configuration de l'app continue même si l'enregistrement GCM ne marche pas.

joanef2 committed:
Merge remote-tracking branch 'origin/master'

# Conflicts:
#	app/src/main/java/org/benindevelopers/ina/Splashscreen.java

joanef2 committed:
- Mise à jour du gradle plugin
- Bug "L'application s'est arretée" lors de la détection de charge corrigé
- Desormais la configuration de l'app continue même si l'enregistrement GCM ne marche pas.

maxxhount committed:
Integration du nouveaus systeme de permission android 6.0

maxxhount committed:
Merge remote-tracking branch 'origin/master'

# Conflicts:
#	app/src/main/java/org/benindevelopers/ina/Splashscreen.java

joanef2 committed:
Quelques mise à jour

maxxhount committed:
Merge remote-tracking branch 'origin/master'

# Conflicts:
#	app/build.gradle
#	app/src/main/java/org/benindevelopers/ina/Splashscreen.java

sethgnavo committed:
Ajout d'un FAB pour notifier de l'état du courant dans sa zone.

joanef2 committed:
Update project to Gradle 2.2

joanef2 committed:
Correction du bug: l'app ne demande pas au user d'activer son GPS
Amélioration du controle de permission pour les android M et supérieurs
